packages/connect-core/src/interceptor.ts

```diff
@@ -54,8 +51,8 @@ export type Interceptor = (next: AnyFn) => AnyFn;
  * response to inspect and log.
  */
 type AnyFn = (
-  req: UnaryRequest | StreamingRequest
-) => Promise<UnaryResponse | StreamingConn>;
+  req: UnaryRequest | StreamRequest
+) => Promise<UnaryResponse | StreamResponse>;
```

Comment:
A streaming function takes a `StreamRequest` and returns the promise for a `StreamResponse`. It's actually the same as a unary function, just that instead of a single message `T`, it's a `AsyncIterable<T>` for streams.